Yosemite/Yosemite/Base/SessionManagerProtocol.swift

```diff
@@ -37,7 +37,7 @@ public protocol SessionManagerProtocol {
 
     /// Default Credentials.
     ///
-    var defaultCredentials: Credentials? { get set}
+    var defaultCredentials: WPCOMCredentials? { get set}
```

Comment:
I think we want to keep `Credentials` as the type here too to support WPOrg credentials later.